api/src/test/java/com/github/dockerjava/api/command/InspectContainerResponseTest.java

```diff
@@ -33,23 +35,23 @@
 
     @Test
     public void roundTrip_full() throws IOException {
-        InspectContainerResponse[] responses = testRoundTrip(CommandJSONSamples.inspectContainerResponse_full,
+        InspectContainerResponse[] responses = JSONTestHelper.testRoundTrip(CommandJSONSamples.inspectContainerResponse_full,
                 InspectContainerResponse[].class);
-        assertEquals(1, responses.length);
+        Assert.assertEquals(1, responses.length);
```

Comment:
Seems like IDE added this full class name after move. If you revert this change (add static import back), diff will be easily to review